### DISCUSSION TITLE
need help to debug / review for two drivers

### DISCUSSION THREAD

**Discussion by bestenfallskongenial** [rsta2/circle#431] [Q&A]
short introduction:
im absolute new in c / c++ and github and i must admid that most concepts are far bejond my understanding,
but i tried to code two drivers, CbuttonDriver and CencoderDriver to, obviously connect 
buttons and rotary encoders to the GPIO of an Raspberry Pi ( in my case a zero V1.3 ) in Circle.

how should i publish the .h and .cpp files for review? 

 ilove to get feedback and help here.
 
 thanks

**Comment by rsta2** [comment]
Can you attach your sources here in a .zip archive? I cannot make promises, 
but I will have a look on it.
